frontend/packages/knative-plugin/src/catalog/event-source-data.ts

```diff
@@ -0,0 +1,45 @@
+import { TFunction } from 'i18next';
+import {
+  EventSourceApiServerModel,
+  EventSourceContainerModel,
+  EventSourceKafkaModel,
+  EventSourcePingModel,
+  EventSourceSinkBindingModel,
+} from '../models';
+
+export const getEventSourceCatalogProviderData = (
```

Comment:
@sahil143 @invincibleJai How do we handle dynamic event sources that are not part of these default event sources here? For example I installed camel k operator and it added Camel source to the catalog but there is no description or provider available for it so it shows blank there - 

![Screenshot from 2020-11-30 22-50-36](https://user-images.githubusercontent.com/6041994/100642580-f1b44e00-335e-11eb-838b-1c96e39c7bb2.png)
![Screenshot from 2020-11-30 22-50-58](https://user-images.githubusercontent.com/6041994/100642586-f37e1180-335e-11eb-9a98-fc8e3ebac69c.png)

Reply:
@rachael-phillips can you help here in case of dynamic source where we don't have any info i.e no description or provider it just looks blank as above cc @sahil143

Reply:
i guess we can't do much here as there is no information , maybe should not show Description title as well

Reply:
Yeah, it makes sense to hide the heading if there's no description.

Reply:
I'm working on getting the description pulled together - thanks for looping me in on this!

Reply:
Hi @rohitkrai03 and @invincibleJai here is the text for Camel Source to be added in a separate PR: 

Generate events from a variety of systems (cloud platforms, social networks, datastores, message brokers, legacy systems, etc.), leveraging components provided by Apache Camel. All Camel Sources use Apache Camel K as the runtime engine.